utils.pwsh/Invoke-DevShell.ps1

```diff
@@ -29,10 +29,12 @@ function Invoke-DevShell {
         . $PSScriptRoot/Utils-Logger.ps1
```

Comment:
While we're here, you might want to change this to import "Logger.ps1" - that line contains the old filename. It just doesn't break as long as you don't import "Invoke-DevShell.ps1" standalone.